api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -125,21 +125,21 @@ public static String getOpenmrsProperty(String property) {
 		return null;
 	}
 		
-	public static String DATABASE_NAME = "openmrs";
+	public static final String DATABASE_NAME = "openmrs";
 	
-	public static String DATABASE_BUSINESS_NAME = "openmrs";
+	public static final String DATABASE_BUSINESS_NAME = "openmrs";
 	
 	/**
 	 * Set true from runtime configuration to obscure patients for system demonstrations
 	 */
-	public static boolean OBSCURE_PATIENTS = false;
+	public static final boolean OBSCURE_PATIENTS = false;
 	
-	public static String OBSCURE_PATIENTS_GIVEN_NAME = "Demo";
+	public static final String OBSCURE_PATIENTS_GIVEN_NAME = "Demo";
 	
-	public static String OBSCURE_PATIENTS_MIDDLE_NAME = null;
-	
-	public static String OBSCURE_PATIENTS_FAMILY_NAME = "Person";
+	public static final String OBSCURE_PATIENTS_MIDDLE_NAME = null;
 	
+	public static final String OBSCURE_PATIENTS_FAMILY_NAME = "Person";
+
```

Comment:
The above fields prefixed with OBSCURE_ are reassigned later

Reply:
@wluyima  OpenmrsConstants.OBSCURE_PATIENTS is modified in PersonNameTest.java. Shall I remove those lines ?